lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,159 @@
+          body: pr_body,
+          head: intermediate_branch,
+          base: base_branch,
+          labels: labels,
+          milestone: milestone
+        )
+      end
+
+      def self.description
+        'Creates backmerge PRs for a release branch into target branches'
+      end
+
+      def self.authors
+        ['Automattic']
+      end
+
+      def self.return_type
+        :array_of_strings
+      end
+
+      def self.return_value
+        'The list of backmerge PRs created'
+      end
+
+      def self.details
+        <<~DETAILS
+          This action creates backmerge Pull Requests from a release branch into one or more target branches.
+
+          It can be used to ensure that changes from a release branch are merged back into other branches, such as newer release branches or the main development branch (e.g., `trunk`).
+        DETAILS
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :repository,
+                                       env_name: 'GHHELPER_REPOSITORY',
+                                       description: 'The remote path of the GH repository on which we work',
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :release_branch,
+                                       description: 'The release branch, in the format `release/x.y.z`',
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :default_branch,
+                                       description: 'The default branch to target if no newer release branches exist',
+                                       optional: true,
+                                       default_value: DEFAULT_BRANCH,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :target_branches,
+                                       description: 'Array of target branches for the backmerge. If empty, the action will determine target branches by finding all `release/x.y.z` branches with a `x.y.z` version greater than the version in source branch's name. If none are found, it will target `default_branch`',
+                                       optional: true,
+                                       default_value: [],
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :labels,
+                                       description: 'The labels that should be assigned to the backmerge PRs',
+                                       optional: true,
+                                       default_value: [],
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :milestone_title,
+                                       description: 'The title of the milestone to assign to the created PRs',
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/release-toolkit/blob/ea252d81f5879173f92a376c4a6dd5638604c1a9/lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb#L147"></span>Lint/Syntax: unterminated string meets end of file<br />
(Using Ruby 3.2 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
  </tbody>
</table>